pkg/scheduler/scheduler.go

```diff
@@ -396,8 +396,9 @@ func (s *Scheduler) doSchedule(key string) error {
 	return s.doScheduleClusterBinding(name)
 }
 
-func (s *Scheduler) doScheduleBinding(namespace, name string) error {
-	rb, err := s.bindingLister.ResourceBindings(namespace).Get(name)
+func (s *Scheduler) doScheduleBinding(namespace, name string) (err error) {
+	var rb *workv1alpha2.ResourceBinding
```

Comment:
lgtm but I'd prefer `:=` than `var`, which seems kinda tedious

Reply:
+1
The named return value already defined the `err` variable, `:=` won't redefine it again.

Reply:
I looked up the grammar, and you're right. I will revert the modification about it